_apps/approveman.md

```diff
@@ -0,0 +1,48 @@
+---
+title: ApproveMan
+description: Approves pull requests with safe changes for protected branches.
+slug: approveman
+screenshots:
+- https://github.com/tianhaoz95/approveman/raw/master/docs/asset/screenshots/approval.png
+- https://github.com/tianhaoz95/approveman/raw/master/docs/asset/screenshots/check_status.png
+authors:
+- tianhaoz95
+repository: tianhaoz95/approveman
+host: https://approveman.herokuapp.com/probot
```

Comment:
Can you remove the `/probot` from `host`?

```suggestion
host: https://approveman.herokuapp.com
```

Reply:
Removed! Thanks!